app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,43 @@
+<nav class="navbar navbar-expand-lg bg-light mb-4">
+  <div class="container">
+   <!-- <a class="navbar-brand" href="#">Navbar</a>-->
+    <%= link_to "Photogram", root_path, class: "navbar-brand" %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <a class="dropdown-item" href="/photos">Photos</a>
```

Comment:
Just one tip, remember now you can start using the ruby helper methods like link_to to help with not having to change the link if it ever changes